src/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV VERSION=%%VERSION%% \
     SHA256SUM=%%SHA256SUM%% \
     OKD_REPO_RELEASES_URL="https://github.com/openshift/origin/releases/download" \
     OC_PLUGINS_REPO="https://github.com/appuio/oc-plugins" \
-    KUBECTL_PLUGINS_PATH="~/.kube/plugins"
+    KUBECTL_PLUGINS_PATH="/opt/kube/plugins"
```

Comment:
Is `/opt/kube/` a good location for the plugins?

I tried to stick to the [default search order](https://docs.openshift.com/container-platform/3.9/cli_reference/extend_cli.html#cli-plugin-loader-search-order), so that installed plugins would be detected automatically.

Reply:
Oh, the problem is based on the fact that we use a different user to build the image that we use when we run it. And we the environment variable we set the location explicitly. Fine. :+1: